migrate.sh

```diff
@@ -61,33 +61,33 @@ do
     boundPVs[${#boundPVs[@]}]=${volumeAttachment}
   done
   if [[ "${#boundPVs[@]}" > "0" ]]; then
-    echo -e "\nThere are still PVC which are using by pods please scale down the application pod and re-run the script\n"
-    echo -e "Cluster ID: "$consumerClusterID
+    echo -e "\n${Red}There are still PVC which are using by pods please scale down the application pod and re-run the script\n${EndColor}"
+    echo -e "${Red}Cluster ID: ${EndColor}"$consumerClusterID
     printf "%s\n" "${boundPVs[@]}"
     exit
   fi
 done
 
-echo "Validation Complete!"
+echo "${Cyan}Validation Complete!${EndColor}"
 
 loginCluster $1 "$backupClusterID"
 
-echo -e "\nTaking a backup of resouces required for backup"
+echo -e "\n${Cyan}Taking a backup of resouces required for backup${EndColor}"
```

Comment:
We can move these echo inside the backupProvider Script like we did for other scripts